src/utilities/plugin-utils.ts

```diff
@@ -42,15 +41,14 @@ export const findUsedApprovedScripts = (activities: Activity[]) => {
           if (!activity.pages[page].sections[section].is_hidden) {
             for (let embeddableNum = 0; embeddableNum < activity.pages[page].sections[section].embeddables.length; embeddableNum++) {
               const embeddable = activity.pages[page].sections[section].embeddables[embeddableNum];
-              if (embeddable.type === "Embeddable::EmbeddablePlugin" && embeddable.plugin?.approved_script_label === "teacherEditionTips" && teacherEditionMode) {
+              if (embeddable.type === "Embeddable::EmbeddablePlugin" && embeddable.plugin) {
```

Comment:
This change might cause there to be extra space when running in non teacher edition mode and viewing a page with with teacher edition items on it.

The issue is that the teacher edition item wrapper (I don't know what this component is actually called) will be added to the page, and then the teacher edition plugin inside of it will decide not render anything because the app isn't in teacher edition mode. That behavior by itself is correct because we are trying to not have these "modes" be explicitly part of the activity player (or lara runtime).  But the problem is that the item wrapper usually has some padding or margins. So even though it has no real content, the padding will still show up in the runtime.

The LARA runtime handles this by observing the content inside of the wrapper and as long as it is empty then no padding or margins are added. The AP doesn't have this observing code yet.

So I'd suggest leaving this code as is for now, perhaps with a comment about what I described above, until something like this observing code is added to the activity player.